addons/im_livechat/static/src/core/channel_invitation_patch.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.discuss.ChannelInvitation" t-inherit-mode="extension">
+        <xpath expr="//*[@name='partnerDetail']" position="replace">
+            <t t-if="props.thread?.type !== 'livechat' or !selectablePartner.lang_name">$0</t>
```

Comment:
How does nullish work in combination with a `!==`? Wouldn't it match when there is no `thread` even though that's not what we want?

Reply:
Actually we want it to match if there is no thread. This if just output `$0` so the template is kept as is. (`undefined !== 'livechat'`)